src/pages/comments/CommentCard.jsx

```diff
@@ -1,4 +1,4 @@
-const CommentCard = (comment) => {
+const CommentCard = ({comment}) => {
```

Comment:
try to make sure you are only completing one task per branch - you've essentially completed tasks 5 (single article) and 6 (comments per article) with this pr